shell/common/rasterizer.h

```diff
@@ -204,6 +204,14 @@ class Rasterizer final : public SnapshotDelegate,
 
   fml::TaskRunnerAffineWeakPtr<SnapshotDelegate> GetSnapshotDelegate() const;
 
+  //----------------------------------------------------------------------------
+  /// @brief      Add a view, implicit or not.
+  void AddView(int64_t view_id);
+
+  //----------------------------------------------------------------------------
+  /// @brief      Remove a view, implicit or not.
```

Comment:
It looks like this file's style is to end with a trailing `///` line:

```suggestion
  /// @brief      Add a view, implicit or not.
  ///
  void AddView(int64_t view_id);

  //----------------------------------------------------------------------------
  /// @brief      Remove a view, implicit or not.
  ///
```